jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java

```diff
@@ -43,9 +30,19 @@
 import org.kie.internal.task.api.model.InternalComment;
 import org.kie.internal.task.api.model.InternalOrganizationalEntity;
 
+import javax.persistence.EntityManagerFactory;
```

Comment:
Is this change of imports based on KIE group code style? If not, I would rather avoid unnecessary formatting changes like these.